core/droidkaigiui/src/commonMain/kotlin/io/github/droidkaigi/confsched/droidkaigiui/architecture/DefaultErrorFallbackContent.kt

```diff
@@ -58,10 +66,19 @@ fun DefaultErrorFallbackContent(modifier: Modifier = Modifier) {
     }
 }
 
-@Preview
+// FIXME Make images change according to light and dark modes.
```

Comment:
Currently, light and dark modes can be displayed separately for colors, but background images cannot be displayed separately.
This is because uiMode does not exist in Preview.